src/types/types.d.ts

```diff
@@ -0,0 +1,52 @@
+export interface User {
```

Comment:
i think it would be more helpful if you rename this to homeTypes or something like that